### ISSUE TITLE
How to make work with AAD v2?

### ISSUE THREAD

**Issue by ghost** [AzureADQuickStarts/WebApp-OpenIDConnect-NodeJS#25]
Hi,

Are you planning to update this example with compatibility to AAD v2?

I tested/tried out registering an app in AAD v2 and got everything set up okay. Then in the configs I replaced the values as instructed but also changed `exports.resourceURL` to `https://graph.microsoft.com`. 

When trying to run the application from the browser things worked until pressing the _login link_ . The MS login screen gives following error: "Sorry, but we’re having trouble signing you in.
AADSTS901002: The 'resource' request parameter is not supported."

So basically this example would need some tuning, have you already looked into this?

BR
Masi

**Comment by ghost** [comment]
Ah, there was a separate example on this
https://github.com/AzureADQuickStarts/AppModelv2-WebApp-OpenIDConnect-nodejs
